easybuild/easyconfigs/c/c-ares/c-ares-1.18.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
```

Comment:
@sassy-crick Any reason to go with `CMakeMake`? Existing easyconfigs for `c-ares` v1.18.1 are using `ConfigureMake`, should we consider changing those as well (if so, let's make that a separate PR)

Reply:
I think that was my wrong. I somehow did not find `c-ares` when I was looking for it so I created a new EC, hence the different description and the Author which I only do when it is a new PR, or a significant change to an existing PR. So when I saw `CMakeList.txt` I decided to go for 'the more modern' `CMake` instead of the more traditional `configure`. 
Given there is already the correct `c-ares` EC in place, we can close this PR.